app/models/middleware_server.rb

```diff
@@ -1,5 +1,6 @@
 class MiddlewareServer < ApplicationRecord
   belongs_to :ext_management_system, :foreign_key => "ems_id"
+  belongs_to :lives_on, :polymorphic => true
```

Comment:
yes, it should remain flexible as it can belong in the future to host, container, etc.